shared/src/main/scala/io/github/hamsters/OptionDefaults.scala

```diff
@@ -1,6 +1,6 @@
 package io.github.hamsters
 
-object OptionHelper {
+object OptionDefaults {
```

Comment:
As it handles empty values, something like EmptyOptionValues would be better IMO.